db/seeds.rb

```diff
@@ -5,3 +5,19 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+
+#Create Items
+Item.create(title: "Possum Stew", description: " A savory Morsel of fur and broth", image: "http://cdn.attackofthecute.com/March-21-2012-01-40-38-fdghnfdgn.jpeg", item_status_id: 1, price: 4.50)
+Item.create(title: "Racoon Ragu", description: " Juicy chunks of fresh racoon tossed in a red sauce", image: "http://upload.wikimedia.org/wikipedia/commons/c/cf/Curious_Raccoon.jpg", item_status_id: 2, price: 9.95)
+
+#Create Users
+User.create(email: 'test@gmail.com', full_name: "test_user", display_name: "test_user", password: "test", role: 0)
```

Comment:
Line is too long. [114/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.